### PR TITLE
Doclint and travis profiles now in parent

### PR DESCRIPTION
**Pull request by massdosage**
I'll replace SNAPSHOT version of parent when HotelsDotCom/hotels-oss-parent#10 is merged and released

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -193,35 +193,4 @@
...
-          <plugin>
-            <groupId>org.eluder.coveralls</groupId>
-            <artifactId>coveralls-maven-plugin</artifactId>
-            <version>4.3.0</version>
-          </plugin>
-        </plugins>
-      </build>
-    </profile>
-    <profile>
-      <id>doclint-java8-disable</id>
```
Comment (ddcprg) [inline pom.xml]: are we not using this?

Comment (massdosage) [reply pom.xml]: We are, it's already in the parent so no need to have it here again.

Comment (ddcprg) [reply pom.xml]: ok, I see

**Comment by ddcprg** [comment]
Add a line in CHANGELOG.md

**Comment by massdosage** [comment]
Since the change isn't user facing or affect waggle-dance itself in any way I didn't think it was worth adding to the changelog.

**Pull request merged**